wpilibj/src/main/java/edu/wpi/first/wpilibj/Filesystem.java

```diff
@@ -41,12 +41,12 @@ public static File getOperatingDirectory() {
   }
 
   /**
-   * Obtains the deploy directory of the program, which is the remote location src/main/deploy is
-   * deployed to by default. On the roboRIO, this is /home/lvuser/deploy. In simulation, it is where
-   * the simulation was launched from, in the subdirectory "src/main/deploy"
+   * Obtains the deployment directory of the program, which is the remote location src/main/deploy
```

Comment:
Since the directory is itself named “deploy”, I think it’s clearer to leave this as is, or maybe just add quotes around “deploy”?  Deployment could mean other things (eg where the executable itself is deployed to).